releases/v20.2.15.md

```diff
@@ -0,0 +1,60 @@
+---
+title: What&#39;s New in v20.2.15
+toc: true
+summary: Additions and changes in CockroachDB version v20.2.15 since version v20.2.14
+---
+
+## August 23, 2021
+
+- For a comprehensive summary of features in v20.2, see the [v20.2 GA release notes](v20.2.0.html).
+- To upgrade to v20.2, see [Upgrade to CockroachDB v20.2](../v20.2/upgrade-cockroach-version.html).
+
+Get future release notes emailed to you:
+
+{% include marketo.html %}
+
+
+### Downloads
+
+<div id="os-tabs" class="filters clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.15.linux-amd64.tgz"><button id="linux" class="filter-button" data-scope="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.15.darwin-10.9-amd64.tgz"><button id="mac" class="filter-button" data-scope="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.15.windows-6.2-amd64.zip"><button id="windows" class="filter-button" data-scope="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.15.src.tgz"><button id="source" class="filter-button" data-scope="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+<section class="filter-content" data-scope="windows">
+{% include windows_warning.md %}
+</section>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach:v20.2.15
+~~~
+
+
+### Enterprise edition changes
+
+- `Kafka_sink_config` can now include `RequiredAcks`. [#69016][#69016]
```

Comment:
Wonder if this should include a reference to changefeeds? 
Like "The `kafka_sink_config` [changefeed](../v20.2/create-changefeed.html) option can now include `RequiredAcks`. [#69016][#69016]"

Reply:
Yup, good idea. TFTR!